netkan/netkan/common.py

```diff
@@ -24,3 +27,15 @@ def sqs_batch_entries(messages: Iterable[Dict[str, str]], batch_size: int = 10)
 def pull_all(repos: Iterable[Repo]) -> None:
     for repo in repos:
         repo.remotes.origin.pull('master', strategy_option='theirs')
+
+
+def github_limit_remaining(token: str) -> int:
+    headers = {
+        'Authorization': f'token {token}',
+        'Content-Type': 'application/json'
+    }
+    response = requests.get(
+        'https://api.github.com/rate_limit',
+        headers=headers,
+    )
+    return response.json()['rate']['remaining']
```

Comment:
The `github` module's method for this is pretty nice!

```suggestion
    return github.Github(token).rate_limiting[0]
```

OR

```suggestion
    return github.Github(token).get_rate_limit().core.remaining
```

Reply:
Good catch! I forgot we were using that library, that's way nicer.